src/librbd/ImageCtx.cc

```diff
@@ -930,7 +930,8 @@ struct C_InvalidateCache : public Context {
         "rbd_journal_object_flush_age", false)(
         "rbd_journal_pool", false)(
         "rbd_journal_max_payload_bytes", false)(
-        "rbd_journal_max_concurrent_object_sets", false);
+        "rbd_journal_max_concurrent_object_sets", false)(
+        "rbd_mirror_resync_after_disconnect", false);
```

Comment:
Valid point -- it still might be a little confusing to have a "per-image" config option named with the same prefix as "rbd_mirror"-specific config options. Perhaps ```rbd_resync_to_primary_mirror_after_disconnect``` or something similar?